.gitignore

```diff
@@ -81,3 +81,5 @@ singletest.sh
 run_with_appinsights.sh
 druncli.sh
 .DS_Store
+
+CosmosKey*
```

Comment:
Newline linter here and in other files too

Reply:
good catch